Maple2.Server.Game/Packets/CubePacket.cs

```diff
@@ -347,7 +347,7 @@ private enum Command : byte {
         return pWriter;
     }
 
-    public static ByteWriter DesignRankReward() {
+    public static ByteWriter DesignRankReward(Home home) {
```

Comment:
why is home passed in?

Reply:
i was going to implement then the pr got too big